api/src/certification/complementary-certification/domain/models/ComplementaryCertificationForAdmin.js

```diff
@@ -0,0 +1,9 @@
+class ComplementaryCertificationForAdmin {
```

Comment:
Dans le sous-contexte actuel, il n'y a pas de model `ComplementaryCertification.js`. Nous n'avons donc aucun raison de le formaliser comme `ForAdmin`.

On pourra se poser la question de quand ça arrivera, pour cela, j'en ai déjà discuter avec Simon (mais ça sera quand le moment sera venu).

```suggestion
class ComplementaryCertification {
```

Reply:
En prenant les retours avec Jerem, on s'est dit `ComplementaryCertification` serait trop large, on a préféré spécifier l'usage de ce dernier pour le lier à l'attachement des profils cible

Reply:
On instancie un model partiel de complementary-certifications, ça n'aurait pas de sens de le nommer juste "ComplementaryCertification".

Le risque serait de le rajouter des attribus manquants dans le futur et de retomber dans les travers de l'ancienne archi